packages/react/src/components/ImpactBadge/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef } from 'react';
+
+import Badge, { BadgeLabel, BadgeProps } from '../Badge';
+
+import { ContentNode } from '../../types';
+import Icon, { IconType } from '../Icon';
+import classNames from 'classnames';
+
+type ImpactType = 'critical' | 'serious' | 'moderate' | 'minor';
+
+interface ImpactBadgeProps extends Omit<BadgeProps, 'children'> {
+  type: ImpactType;
+  label?: ContentNode;
+}
+
+const iconByType: { [type in ImpactType]: IconType } = {
+  critical: 'chevron-double-up',
+  serious: 'chevron-up',
+  moderate: 'chevron-down',
+  minor: 'chevron-double-down'
+};
+
+const typeValues: { [key in ImpactType]: string } = {
+  critical: 'Critical',
+  serious: 'Serious',
+  moderate: 'Moderate',
+  minor: 'Minor'
+};
+
+const ImpactBadge = forwardRef<HTMLDivElement, ImpactBadgeProps>(
+  ({ type, label, className, ...other }, ref) => {
+    return (
+      <Badge
+        className={classNames(`ImpactBadge`, `ImpactBadge--${type}`, className)}
+        ref={ref}
+        {...other}
+      >
+        <Icon type={iconByType[type]} />
+        <BadgeLabel>{label || 'Impact:'}</BadgeLabel>
+        {typeValues[type]}
```

Comment:
This isn't quite right. We need to be able to pass in the label/value collectively for translations elsewhere - the way this is formatted cannot be translated correctly.

This should look something like:

```tsx
<Badge {...props}>
  <Icon {...} />
  {label || (
    <>
      <BadgeLabel>Impact:</BadgeLabel>
      {typeValues[type]}
    </>
  )}
</Badge>
```